timed/reports/tests/test_work_report.py

```diff
@@ -21,7 +21,12 @@ def test_work_report_single_project(auth_client, django_assert_num_queries):
     project = ProjectFactory.create(customer=customer, name="Project/")
     task = TaskFactory.create(project=project)
     ReportFactory.create_batch(
-        10, user=user, verified_by=user, task=task, date=date(2017, 8, 17)
+        10,
```

Comment:
I suggest creating some billable ones, and some non-billables, so we have both to test.